CONTRIBUTING.md

```diff
@@ -89,6 +89,16 @@ or (`DEBUG=true` enables printing stack traces when errors are thrown):
 DEBUG=true ./bin/run.js [command]
 ```
 
+### Attaching a Debugger
+
+When debugging a project, it's super helpful to attach a debugger to the CLI. If you use VS Code, here's how you can do it:
+
+1. Open this repository in VS Code.
+2. Open a "Javascript Debug Terminal" (e.g. by searching for it in the Command Palette (Shift-Cmd+P)). Every Node process that's opened in this terminal will have a debugger attached.
+3. Place a breakpoint somewhere in the CLI. You might have to place them in the compiled `.js` files as opposed to the `.ts` files.
```

Comment:
Currently, no they will never work. If we enabled source maps though, it could! Let's clarify this for now.
```suggestion
3. Place a breakpoint somewhere in the CLI. You will have to place them in the compiled `.js` files as opposed to the `.ts` files.
```